Dockerfile.rpm

```diff
@@ -0,0 +1,43 @@
+FROM redhat/ubi8
```

Comment:
Shouldn't this be `ubi8/ubi-minimal` https://catalog.redhat.com/software/containers/ubi8/ubi-minimal/5c359a62bed8bd75a2c3fba8 ? Or is that what you meant by we cannot do official Redhat images?

Reply:
it should not. minimal utilizes a different installation method we don't currently support